marsha/core/tests/factories.py

```diff
@@ -0,0 +1,93 @@
+"""Factories for the ``core`` app of the Marsha project."""
+
+
+import factory
+
+from marsha.core import models
+
+
+class BaseSoftDeleteFactory(factory.django.DjangoModelFactory):
+    """Base factory for all soft-deletable models."""
+
+    class Meta:  # noqa
+        abstract = True
+
+    deleted = False
```

Comment:
Why do you need a factory for this if the default is None?

Reply:
I better enforce this. But I can remove this base factory if you want.

Reply:
I think you should not enforce it in your tests because you might hide a bug (like a wrong default on deleted?). You should let it be the way it will be on a model when it is created in production...

IMO the good practice in factories is to either let the field have its default value or set a random value to prove that your tests work whatever the user sets... Faker provides realistic random values that a user might set for each type of field.